nf_core/pipeline-template/main.nf

```diff
@@ -86,6 +86,8 @@ workflow {
         params.input
     )
 
+    params.max_time == 42
```

Comment:
Instead of modifying the pipeline-template, which will add this change to all nf-core pipelines, we should modify this file within the test.

Reply:
oh, I didn't realise that. I'll add a line on-the-fly instead.